examples/system/append-inline-properties-to-system-configuration.sql

```diff
@@ -0,0 +1,66 @@
+-- ============================================================================
+
+-- Description:
+-- In this query, make properties available to Luminesce.
+-- This approach appends rather than overwrites the current inline property configuration
+-- See the following page for further details:
+-- https://support.lusid.com/knowledgebase/article/KA-01702/en-us
+
+-- ============================================================================
+
+-- 1. Get the contents of the current instrument provider factory
+
+@@currentFileContent = 
+select Content from Sys.File where Name = 'instrumentsimpleinstrumentproviderfactory';
+
+@currentFileLines = 
+select Value from Tools.Split
+where Original = @@currentFileContent
+    and Delimiters = '
+' -- newline
+    and [Index] <> 1 -- skip header
+    and Value is not null
+    and Trim(Value, ' ') <> '';
+
+@splitFileLines =
+select OriginalIndex, 
+    case when [Index] = 1 then Value else null end as PropertyKey, 
+    case when [Index] = 2 then Value else null end as DataType, 
+    case when [Index] = 3 then Value else null end as Alias
+from Tools.Split
+where Original in @currentFileLines and Delimiters = ',';

```

Comment:
This will all work fine - however we have recently added support for adding descriptions to inlined columns.
So l25-31 would currently omit this description, and not preserve the content of the inlining file.

Just adding another case ought to work, ie:
```
case when [Index] = 3 then Value else null end as Alias,
case when [Index] = 4 then Value else null end as Description
```

and similar changes below